content/release-10-10-0/oee-10-10-0-bundle/10_10_0_3.md

```diff
@@ -0,0 +1,187 @@
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">Until now, all profiles needed a 10 min (600s) default calculation interval. If this interval was not configured for the calculation profile, it was automatically added by OEE application. With this change, it is now possible to define a default calculation interval for each profile, which will be used on the machine park overview. Calculation profiles created before this change, will be migrated to use the 10 min interval as their default interval.<br/><br/>With this change it is now also possible to reduce the number of MEAs created for the calculcation profiles. For slow producing machines, it might for example not be required to create a measurement every 10 mins.</td>
+<td style="text-align:left">COE-443</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">Rule validation has been significantly improved for the <i>Matching</i> dialogs in the profile editor. The new validation now evaluates the type of calculation step to derive the type of the output of the whole formula. This type is compared with the expected output and errors are reported inline in the rule editor.</td>
+<td style="text-align:left">COE-441<br/>COE-508</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">It is now allowed to assign machines to multiple organisational units (such as Lines, Areas and Sites). The dialog for assigning (adding) machines to an orgisational entity now lists all unassigned machines before listing the machines already assigned to any other entity. Parent organisational entities of all assigned machines are presented in the entry selection.</td>
+<td style="text-align:left">COE-547</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">Drag-and-drop in <i>Organisation</i> <i>Admin</i> section has been enabled by default, but is now limited to reordering entities within it's parent entity. The switch to enable drag-and-drop in the list header has been removed.</td>
+<td style="text-align:left">COE-609</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">In the machine dashboard, showing alarms in the OEE data chart can now be enabled and disabled. To enable and disable alarms in the chart, a new switch was added at the bottom of the <i>Legend</i> of the chart. In addition the number of alarms loaded in the machine dashboard was optimized to improve performance.</td>
+<td style="text-align:left">COE-338<br/>COE-499<br/>COE-510</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">When choosing the calculation method in the <i>Matching</i> dialog of the profile editor, the entire tile can now be clicked to select a computation method.</td>
```

Comment:
Calculation methods vs. computation method. Use the same term in both locations.